startpage_content.md

```diff
@@ -0,0 +1,62 @@
+This WIP PR is for preparing content for the start page.
+
+Please commit your ideas until 2019-05-11 so that we can discuss on the next web meeting about the design.
+
+Solutons could be:
+
+* screens (can take a lot of content)
+* carousel/slides (all at top, do not take much content)
+* carousel/slides with screens (teaser + _learn more_-button)
+* ...
+
+# Persistent content
+
+## Green Open Access
+_(Citation advantage of OA https://sparceurope.org/what-we-do/open-access/sparc-europe-open-access-resources/open-access-citation-advantage-service-oaca/ )_
+
+Many researchers do not use their right to make their papers freely available online, in addition to the paywalled version offered by traditional publishers.
+
+There are two major advantages of unpaywalling your papers:
+1. There are a lot of studies giving evidence, that freely available publications have a higher citation rate. 
+This is called the _Open Access Citation Advantage_. You can find a list of studies on https://sparceurope.org/what-we-do/open-access/sparc-europe-open-access-resources/open-access-citation-advantage-service-oaca/oaca-list/ 
+As a conclusion: Your research is not only more acknowledged, but pushes scientific advantage in general, because your research better spreads!
+1. Libraries do not need to by overpriced electronic subscriptions to journals, when they can afford them at all.
+
+## Open repositories
+Uploading your papers on your own webpage is not enough. Such copies are less stable and harder to find than documents uploaded to well-indexed repositories.
+There are for a lot of major repositories like arXiv, chemRxiv and institutional repositories. Nearly all of them are searched by BASE and CORE. And they ensure a long-term availability of your research.
+
+Dissemin searches for copies of your papers in a large collection of open repositories and tells you which ones cannot be accessed. 
+
+## Statistics
+_(With d3js it sould be easy to create some fancy, dynamic on-the-fly-graphs)_
+
+Since __%date__ __%number__ of publications have been uploaded via dissemin. This make __%number__ publications per week.
+_%time series graph_
+
+## Success-Story
+_(Get some data from zenodo, maybe contact the depositor?)_
+
+The publication __%publication__ was uploaded on __%date__ and got __%number__ downloads and __%number__ views on Zenodo!
+
+The depositor says: Uploading this publication on dissem.in was the best idea I ever had!
+
+We have another __%number__ publications with at least __%number__downloads on Zenodo.
+
+
+## Dissemin for libraries
```

Comment:
I doubt that libraries are a large proportion of our visitors, so I think this should maybe be a separate page. There would be something like "are you a library? Discover here how you can use Dissemin" with a link.

I also think the text should be clearer, in particular separating what we offer ("you can use...") vs what we ask ("you can advertize..." should probably be "advertise" btw). "connect your repository" should be explained better and is arguably targeted towards other people, namely, repository admins.

Maybe we should have a list of links for different audiences, like:

- Are you a researcher? Discover how to start depositing your papers with Dissemin
- Are you a librarian? (or "managing a lab"?) Discover how to use Dissemin to find the papers from your institution
- Are you a repository administrator? Discover how to index your repository in Dissemin
- Are you a developer? Discover how to use our API